frontend/src/components/Item/ItemMenu.vue

```diff
@@ -36,6 +36,10 @@
     v-if="item.Id"
     v-model:dialog="metadataDialog"
     :item-id="item.Id" />
+  <identify-dialog
+    v-if="item.Id"
+    v-model:dialog="identifyItemDialog"
+    :item-id="item.Id" />
```

Comment:
Please match the callback style I used for the refreshmetadata dialog, to ensure only strictly necessary components are on screen all the time.